website/addons/dryad/model.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+
+from modularodm import fields
+
+from framework.exceptions import HTTPError
+
+from website.addons.base import AddonUserSettingsBase, AddonNodeSettingsBase
+from website.addons.base import StorageAddonBase
+
+from website.addons.dryad import settings as dryad_settings
+#from website.addons.dryad import serializer as serializer
+
+from pyDryad import Dryad
+
+
+class AddonDryadUserSettings(AddonUserSettingsBase):
```

Comment:
I saw this class definition and was at first very confused. 

Do you think you could add a docstring to this class briefly explaining the Dryad API's authentication model (or lack thereof)?